openshift/oadp-operator
# [oadp-1.4] fix: automate upgrade tests

## Why the changes were made

Automate upgrade tests process, with focus in 1.4.0. (master branch PR https://github.com/openshift/oadp-operator/pull/1446)

Also, cherry-pick of https://github.com/openshift/oadp-operator/pull/1463

## How to test the changes made

Follow [E2E testing documentation](https://github.com/mateusoliveira43/oadp-operator/blob/fix/automate-test-upgrade/docs/developer/testing/TESTING.md) and run
```sh
make catalog-test-upgrade
TEST_UPGRADE=true make test-e2e
```

To run tests manually, check [upgrade testing documentation](https://github.com/mateusoliveira43/oadp-operator/blob/fix/automate-test-upgrade/docs/developer/testing/test_oadp_version_upgrade.md).
